tests/parser/GroupTests.cpp

```diff
@@ -442,3 +427,79 @@ BOOST_AUTO_TEST_CASE(TESTGCONSALE) {
 
 
 }
+
+BOOST_AUTO_TEST_CASE(GCONINJE_MULTIPLE_PHASES) {
+    Parser parser;
+    std::string input = R"(
+        START             -- 0
+        31 AUG 1993 /
+        SCHEDULE
+
+        GRUPTREE
+           'G1'  'FIELD' /
+           'G2'  'FIELD' /
+        /
+
+        GCONINJE
+           'G1'   'WATER'   1*  1000      /
+           'G1'   'GAS'     1*  1*   2000 /
+           'G2'   'WATER'   1*  1000      /
+        /
+
+        TSTEP
+           10 /
+
+        GCONINJE
+           'G2'   'WATER'   1*  1000      /
+           'G2'   'GAS'     1*  1*   2000 /
+           'G1'   'GAS'     1*  1000      /
+        /
+
+        )";
+
+    auto deck = parser.parseString(input);
+    EclipseGrid grid(10,10,10);
+    TableManager table ( deck );
+    FieldPropsManager fp( deck , grid, table);
```

Comment:
This `FieldPropsManager` constructor call is no longer valid.  PR #1417 added a new parameter&mdash;the run's set of active phases&mdash;to the parameter list.

This PR must be updated accordingly.

Reply:
Ok - I can not fix this right now, @totto82 - if you want to steal this and get it in asap please do so, otherwise I will eventually pick it up.

Reply:
I only have time after the opm meeting.

Reply:
@joakim-hove Are you on this?

Reply:
Yes - this has been fixed and rebased "long time ago"